.travis.yml

```diff
@@ -48,7 +48,7 @@ install:
   - phpenv config-rm xdebug.ini
 
   # The installation will include wp-cli as well as it's a requirement of wp-browser.
-  - composer update --prefer-dist
+  - composer install
```

Comment:
@borkweb wasn't there a reason why we do `composer update` rather than `composer install`? Is that relevant here? @bordoni you might know?

Reply:
I've set that up. The reason we did `composer update` was as we did commit `composer.lock` files generated on different machines and PHP versions from PHP 7.0.

Since, now, we have `tric` to make sure `composer ...` operations will take place in a PHP 7.0 context, there's no reason to pay the overhead time to update in tests.

Reply:
One typical scenario we did run into was someone doing `composer update` on a PHP 7.3 machine, committing the lock file and having the wrong dependencies pulled in CI, where PHP 7.0 is used.